metapool/prep.py

```diff
@@ -10,10 +10,17 @@
 
 # TODO: Ideally these should be imported from Qiita
 
-REQUIRED_COLUMNS = {'well_description', 'sample_plate', 'sample_well',
-                    'i7_index_id', 'index', 'i5_index_id', 'index2'}
+# well_description is no longer a required column, since the sample-name
```

Comment:
I think this description would be nice to have in the lines of code that actually do this, not on the top of the file - could you move?

Reply:
Will do! I was thinking the same thing.